### PR TITLE
Rename "private messages" to "direct messages"

### LINKED ISSUE #8970: Rename "PM" to "Message" or similar
## Problem

When a player receives a message sent with `/msg`, it is displayed as `"PM from <user>: <content>."`
"Private message" is a misleading name for this, as PMs are logged prominently in debug.txt. It's debatable how important this issue is, but in extreme cases one could even encounter legal issues for looking at these "private" messages while scrolling through debug.txt (citation needed, I'm not a European and much less a lawyer.)

## Solutions

- Rename "PM" to "Message" or "Direct Message"

## Alternatives

- Not logging PMs at all, or letting players agree to disclose their PMs (inadvisable, because the messages are invaluable for moderation.)

### PR DESCRIPTION
**Pull request by Calinou**
These messages can be read by server administrators, which makes them not actually private.

This closes #8970.

## To do

This PR is Ready for Review.

## How to test

Start a local game, type `/help msg` to check the command description, then send a message to yourself.

### CODE REVIEW

**Comment by ClobberXD** [comment]
:+1:

File: builtin/game/chat.lua [hunk 1]
```diff
@@ -990,7 +990,7 @@ core.register_chatcommand("clearobjects", {
 
 core.register_chatcommand("msg", {
 	params = "<name> <message>",
-	description = "Send a private message",
+	description = "Send a message directly to a player",
```
Comment (ClobberXD) [inline builtin/game/chat.lua]: This doesn't sound right. Maybe this?

```suggestion
	description = "Send a direct message to a player",
```

or maybe even just

```suggestion
	description = "Send a message to a player",
```

`to a player` implies that it's direct.

**Comment by IhrFussel** [comment]
Are you sure that "Messages" is not confusing?

Because ALL messages in the chat are "messages" whether they are public ones, channel ones, server ones or global ones...

**Comment by ClobberXD** [comment]
IMO, `Message from <player>:` makes it clear that this is a message from one player to another, and not a global chat-message.

**Comment by IhrFussel** [comment]
That is not the point.

The point is how should we server owners and players in general refer to 'PM' if they are just called 'messages' now?

"Hey if you have any questions just send me a message"

**Comment by jastevenson303** [comment]
Wat about 'private message' .. we could abbreviate it as .. oh wait

**Comment by Calinou** [comment]
> The point is how should we server owners and players in general refer to 'PM' if they are just called 'messages' now?

I guess we should rename them to direct messages then :wink:

**Comment by paramat** [comment]
Hmm well, the context in code, in command description and in logging makes it clear it is a personal message, i think the PR is fine as it is.
People can just refer to it whatever way they want to make clear it is a DM/PM/whatever.

**Comment by paramat** [comment]
I don't like 'Direct message' so am neutral.

**Pull request merged**